doc/source/release-notes/Percona-Server-5.5.44-37.3.rst

```diff
@@ -0,0 +1,28 @@
+.. rn:: 5.5.44-37.3
+
+==============================
+ |Percona Server| 5.5.44-37.3
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.44-37.3 on June 29th, 2015. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.44-37.3/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.44 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-44.html>`_, including all the bug fixes in it, |Percona Server| 5.5.44-37.3 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.44-37.3 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.44-37.3>`_. 
+
+Bugs Fixed
+==========
+
+ Symlinks to ``libmysqlclient`` libraries were missing on *CentOS* 6. Bug fixed :bug:`1408500`.
+
+ *RHEL/CentOS* 6.6 ``openssl`` package (1.0.1e-30.el6_6.9), containing a fix for CVE-2015-4000, changed the DH key sizes to a minimum of 768 bits. This caused an issue for *MySQL* as it uses 512 bit key by default. Bug fixed :bug:`1462856` (upstream :mysqlbug:`77275`).
+
+ *innochecksum* would fail to check table files when compressed row format with ``KEY_BLOCK_SIZE`` set to ``4`` was used. Fix for this bug has been ported from *Facebook MySQL* 5.1 patch. Bug fixed :bug:`1100652` (upstream :bug:`66779`).
```

Comment:
innochecksum would fail to check tablespaces in compressed format. The fix for this bug has been ...